src/global/common.rs

```diff
@@ -633,14 +636,74 @@ impl StateChanges {
 
                         if uninstalled_items.len() == 1 {
                             eprintln!(
-                                "{}Uninstalled shortcut {} in environment {}.",
+                                "{}Uninstalled shortcut {} of environment {}.",
                                 console::style(console::Emoji("✔ ", "")).green(),
                                 uninstalled_items[0],
                                 env_name.fancy_display()
                             );
                         } else {
                             eprintln!(
-                                "{}Uninstalled shortcuts in environment {}:",
+                                "{}Uninstalled shortcuts of environment {}:",
+                                console::style(console::Emoji("✔ ", "")).green(),
+                                env_name.fancy_display()
+                            );
+                            for uninstalled_item in uninstalled_items {
+                                eprintln!("   - {}", uninstalled_item);
+                            }
+                        }
+                    }
+                    StateChange::AddedCompletion(name) => {
+                        let mut installed_items = StateChanges::accumulate_changes(
+                            &mut iter,
+                            |next| match next {
+                                Some(StateChange::InstalledShortcut(name)) => Some(name.clone()),
```

Comment:
Is this related to shortcuts?

Reply:
No, stupid auto-complete mistake. Nice find!